src/firebase.js

```diff
@@ -44,6 +45,43 @@ const initParticipant = (participantId, studyId, startDate) => {
     });
 };
 
+// get config from firebase
+const getFirestoreConfig = (studyID, docName) => {
```

Comment:
most of the other functions here are written in the `async / await` style instead of `.then .catch` - consider switching it over